ddl/get.lua

```diff
@@ -52,12 +52,31 @@ local function _get_index(box_space, box_index)
     return ddl_index
 end
 
-local function get_space_sharding_key(space_name)
-    if box.space._ddl_sharding_key == nil then
+local function get_metadata(space_name, metadata_name)
+    local metadata_space_name = string.format("_ddl_%s", metadata_name)
+
+    if box.space[metadata_space_name] == nil then
+        return nil
+    end
+
+    return box.space[metadata_space_name]:get{space_name}
+end
```

Comment:
It would be good to think about cache metadata and its invalidation on schema update.

Reply:
Updated

Reply:
Cache for metadata was removed for some reasons that was described by @Totktonada (https://github.com/tarantool/ddl/pull/72#discussion_r704860471)